internal/provider/resources/work_pool.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/nsf/jsondiff"
```

Comment:
[This package](https://github.com/nsf/jsondiff) only has a couple hundred stars, but it gave us the ability to return the differences in the output so we don't have to return the entire lengthy base job templates which is tough to debug (more detail in https://github.com/PrefectHQ/terraform-provider-prefect/pull/249/commits/9f67474fbc35cf4034b0d1f0f21bbadf693f2316).

If we prefer, we can go back to [google/go-cmp](https://github.com/google/go-cmp) which has several thousand stars and is better maintained. (This just means reverting to https://github.com/PrefectHQ/terraform-provider-prefect/pull/249/commits/810013f8d47a8807b3c6001e643b54b2ddca7375)

Reply:
Going to give https://github.com/go-test/deep a try tomorrow.

Reply:
This worked out better. Pushed 2d627e538bb5cc94732c418a593a0904c70e7c62